### ISSUE TITLE
Add perceptual loss from pre-trained CNN

### ISSUE THREAD

**Issue by RobinCSIRO** [tadax/glcic#7]
Hi, @tadax, thank you very much for your project. I am new to the task of 'image inpainting', and I noticed that most of papers of this project adopt another loss, called perceptual loss by calculating the difference of reconstructed image and original image with the feature vector extracted from a pre-trained CNN (e.g., VGG16). Could you please show me how to add such loss into your model or recommend me some other implementation you know to realize such tasks? Thank you very much~~

**Comment by Abhi001vj** [comment]
may be there is no need to use another model for perceptual loss as this network has two discriminators which are doing this already if I understand this correctly . Thats what i thought when I looked at the abstract for the project in the website for the paper.
